lib/static/modules/action-names.ts

```diff
@@ -19,6 +17,11 @@ export default {
     SCREENSHOT_ACCEPTER_ACCEPT: 'SCREENSHOT_ACCEPTER/ACCEPT',
     APPLY_DELAYED_TEST_RESULTS: 'APPLY_DELAYED_TEST_RESULTS',
     UNDO_ACCEPT_IMAGES: 'UNDO_ACCEPT_IMAGES',
+    STATIC_ACCEPTER_DELAY_SCREENSHOT: 'STATIC_ACCEPTER_DELAY_SCREENSHOT',
+    STATIC_ACCEPTER_UNDO_DELAY_SCREENSHOT: 'STATIC_ACCEPTER_UNDO_DELAY_SCREENSHOT',
+    STATIC_ACCEPTER_STAGE_SCREENSHOT: 'STATIC_ACCEPTER_STAGE_SCREENSHOT',
+    STATIC_ACCEPTER_UNSTAGE_SCREENSHOT: 'STATIC_ACCEPTER_UNSTAGE_SCREENSHOT',
+    STATIC_ACCEPTER_COMMIT_SCREENSHOT: 'STATIC_ACCEPTER_COMMIT_SCREENSHOT',
```

Comment:
> STATIC_ACCEPTER_DELAY_SCREENSHOT

screenshot accepter only. Triggers on "Accept". Does not mutate "tree" state, therefore, it is faster. These would be staged on screenshot accepter exit (it would trigger tree update)

> STATIC_ACCEPTER_UNDO_DELAY_SCREENSHOT

screenshot accepter only. Removes last accepted delayed screenshots.

> STATIC_ACCEPTER_STAGE_SCREENSHOT

is called from the report body and on screenshot accepter exit

> STATIC_ACCEPTER_UNSTAGE_SCREENSHOT

is called from the report body. Unstaged screenshot from commit

> STATIC_ACCEPTER_COMMIT_SCREENSHOT

commits staged (and delayed) screenshots